assembly-multiuser/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/multiuser.properties

```diff
@@ -8,72 +8,62 @@
 #     suspend the workspace by snapshotting it and then stopping it. Idleness is the
 #     length of time that the user has not interacted with the workspace, meaning that
 #     one of our agents has not received interaction. Leaving a browser window open
 #     counts toward idleness.
 che.limits.workspace.idle.timeout=-1
-
 #####                             USERS' WORKSPACE LIMITS                          #####
-
 #     The total amount of RAM that a single user is allowed to allocate to running
 #     workspaces. A user can allocate this RAM to a single workspace or spread it
 #     across multiple workspaces.
 che.limits.user.workspaces.ram=-1
-
 #     The maximum number of workspaces that a user is allowed to create. The user will
 #     be presented with an error message if they try to create additional workspaces.
 #     This applies to the total number of both running and stopped workspaces. Since
 #     each workspace is saved as a snapshot, placing a cap on this number is a way
 #     to limit the disk consumption for workspace storage.
 che.limits.user.workspaces.count=-1
-
 #     The maximum number of running workspaces that a single user is allowed to have.
 #     If the user has reached this threshold and they try to start an additional
 #     workspace, they will be prompted with an error message. The user will need to
 #     stop a running workspace to activate another.
 che.limits.user.workspaces.run.count=-1
-
 #####                             ORGANIZATIONS' WORKSPACE LIMITS                  #####
-
 #     The total amount of RAM that a single organization (team) is allowed to allocate
 #     to running workspaces. An organization owner can allocate this RAM however they
 #     see fit across the team's workspaces.
 che.limits.organization.workspaces.ram=-1
-
 #     The maximum number of workspaces that a organization is allowed to own. The
 #     organization will be presented an error message if they try to create
 #     additional workspaces. This applies to the total number of both running
 #     and stopped workspaces. Since each workspace is saved as a snapshot, placing a
 #     cap on this number limits the disk consumption for workspace storage.
-
 che.limits.organization.workspaces.count=-1
 #     The maximum number of running workspaces that a single organization is allowed.
 #     If the organization has reached this threshold and they try to start an
 #     additional workspace, they will be prompted with an error message. The
 #     organization will need to stop a running workspace to activate another.
 che.limits.organization.workspaces.run.count=-1
+#####                             KEYCLOAK                                        #####
+#     Grant system permission for 'che.admin.name' user. If the user already exists it'll happen on
+#     component startup, if not - during the first login when user is persisted in the database.
+che.admin.name=admin
```

Comment:
I'm not sure that it should be in `KEYCLOAK` section. I'd put it below `CHE SYSTEM` section and rename this property to something like `che.system.admin_name`